app/user/Makefile

```diff
@@ -38,13 +38,7 @@ DEFINES += -DESP_INIT_DATA_DEFAULT="\"$(SDK_DIR)/bin/esp_init_data_default_v05.b
 # Required for each makefile to inherit from the parent
 #
 
-INCLUDES := $(INCLUDES) -I $(PDIR)include
-INCLUDES += -I ./
-INCLUDES += -I ../../include/ets
-INCLUDES += -I ../libc
-INCLUDES += -I ../platform
-INCLUDES += -I ../lua
-INCLUDES += -I ../spiffs
+INCLUDES += -I ../../include/ets -I ../spiffs
```

Comment:
I don't think we have an `include/ets` either, we should be able to evict that too.

Reply:
Will do